modules/administration-guide/partials/proc_applying-the-bitbucket-application-link-secret.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: PROCEDURE
+:description: Applying the Bitbucket application link Secret
+:keywords: applying-bitbucket, apply-the-bitbucket, apply-bitbucket, apply-secret, applying-secret, apply-a-secret, applying-a-secret, bitbucket, bitbucket-application, bitbucket-app, bitbucket-application-link, bitbucket-app-link, bitbucket-secret, secret
+:navtitle: Applying the Bitbucket application link Secret
+// :page-aliases:
+
+[id="applying-the-bitbucket-application-link-secret_{context}"]
+= Applying the Bitbucket application link Secret
+
+Prepare and apply the Bitbucket application link Secret as follows:
+
+.Prerequisites
+* The preceding procedure, Setting up the Bitbucket application link, has been completed.
+* The following Base64-encoded files that were created during the preceding procedure are prepared:
+** `privatepkcs8-stripped.pem`
+** `bitbucket_consumer_key`
+** `bitbucket-shared-secret`
+* `{orch-cli}` is installed in the operating system you are using.
+////
+{orch-cli}=oc
+https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli
+https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
+////
+
+.Procedure
+
+. Prepare the secret like so:
++
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: Secret
+apiVersion: v1
+metadata:
+  name: bitbucket-oauth-config
+  namespace: {prod-namespace} <1>
+  labels:
+    app.kubernetes.io/component: oauth-scm-configuration
+    app.kubernetes.io/part-of: che.eclipse.org
+  annotations:
+    che.eclipse.org/oauth-scm-server: bitbucket
+    che.eclipse.org/scm-server-endpoint: __<bitbucket_server_url>__ <2>
+type: Opaque
+data:
+  private.key: __<base64_content_of_privatepkcs8-stripped.pem>__ <3>
+  consumer.key: __<base64_content_of_bitbucket_server_consumer_key>__ <4>
+  shared_secret: __<base64_content_of_bitbucket-shared-secret>__ <5>
+----
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
```

Comment:
Do your build tools render this instance of `{prod-namespace}` correctly?

Reply:
Yes. Thank you for checking.